build_mt.py

```diff
@@ -26,7 +26,7 @@
 from astropy.io import ascii
 
 from tools.InputFile import InputFile
-from tools.merger import simple_load, get_slab_halo, extract_superslab, extract_superslab_minified
+from tools.merger import simple_load, get_halos_per_slab, extract_superslab, extract_superslab_minified
```

Comment:
Renamed `get_slab_halo()` to `get_halos_per_slab()`; just trying to be more explicit about what the function does.